storage/uri.go

```diff
@@ -211,6 +211,42 @@ func Delete(u fyne.URI) error {
 
 }
 
+func DeleteRecursive(u fyne.URI) error {
+	list, err := List(u)
+
```

Comment:
Please remove lines like this between the function call and checking errors. This isn't idiomatic Go code. Same applies in more places.